vdev/src/testing/runner.rs

```diff
@@ -205,28 +210,35 @@ pub trait ContainerTestRunner: TestRunner {
     }
 
     fn create(&self) -> Result<()> {
-        dockercmd([
-            "create",
-            "--name",
-            &self.container_name(),
-            "--network",
-            &self.network_name(),
-            "--hostname",
-            RUNNER_HOSTNAME,
-            "--workdir",
-            MOUNT_PATH,
-            "--volume",
-            &format!("{}:{MOUNT_PATH}", app::path()),
-            "--volume",
-            &format!("{VOLUME_TARGET}:{TARGET_PATH}"),
-            "--volume",
-            &format!("{VOLUME_CARGO_GIT}:/usr/local/cargo/git"),
-            "--volume",
-            &format!("{VOLUME_CARGO_REGISTRY}:/usr/local/cargo/registry"),
-            &self.image_name(),
-            "/bin/sleep",
-            "infinity",
-        ])
+        let network_args = match self.network_name() {
+            Some(name) => vec!["--network".into(), name],
+            None => vec![],
+        };
+        let docker_volume = format!("{}:/var/run/docker.sock", DOCKER_SOCK.display());
```

Comment:
Small nit but the name could be better descriptive as "docker volume" is an existing concept and used several times below. Maybe just `docker_sock`?

Reply:
Fixed